src/test/java/seedu/address/logic/commands/HelpCommandTest.java

```diff
@@ -1,20 +1,22 @@
 package seedu.address.logic.commands;
 
-//import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-/*import static seedu.address.logic.commands.HelpCommand.SHOWING_HELP_MESSAGE;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.HelpCommand.SHOWING_HELP_MESSAGE;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.logic.commands.results.CommandResult;
 import seedu.address.model.Model;
-import seedu.address.model.ModelManager;*/
+import seedu.address.model.ModelManager;
 
 public class HelpCommandTest {
-//    private Model model = new ModelManager();
-//    private Model expectedModel = new ModelManager();
-//
-//    @Test
-//    public void execute_help_success() {
-//        CommandResult expectedCommandResult = new CommandResult(SHOWING_HELP_MESSAGE, true, false);
-//        assertCommandSuccess(new HelpCommand(), model, expectedCommandResult, expectedModel);
-//    }
+    private Model model = new ModelManager();
+    private Model expectedModel = new ModelManager();
+
+    @Test
+    public void execute_help_success() {
+        CommandResult expectedCommandResult = new CommandResult("", SHOWING_HELP_MESSAGE,
```

Comment:
Since you're doing the tests can you push the change to the HELP_MESSAGE too? Apparently it's not our UG link.